pkg/cli/cli.go

```diff
@@ -128,10 +128,7 @@ const (
 )
 
 const (
-	ConnectCommandName                  = "connect"
-	DisconnectCommandName               = "disconnect"
-	ConnectionWebRTCAgentCommandName    = "webrtc"
-	ConnectionWebSocketAgentCommandName = "websocket"
+	ConnectionWebRTCAgentCommandName = "webrtc"
```

Comment:
Why is "websocket" used as a string literal while "webrtc" is a constant? Shouldn't those two be similarly used?